test/smoke/packages/cartographer/0.2.2/cartographer-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TCE_REPO_PATH="$(git rev-parse --show-toplevel)"
+# shellcheck source=test/smoke/packages/utils/smoke-tests-utils.sh
+source "${TCE_REPO_PATH}/test/smoke/packages/utils/smoke-tests-utils.sh"
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Checking package is installed or not
+tanzu package installed list | grep "cert-manager.community.tanzu.vmware.com" || {
+    version=$(tanzu package available list cert-manager.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install cert-manager --package-name cert-manager.community.tanzu.vmware.com --version "${version}"
+}
+
+tanzu package installed list | grep "cartographer.community.tanzu.vmware.com" || {
+    version=$(tanzu package available list cartographer.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+    tanzu package install cartographer --package-name cartographer.community.tanzu.vmware.com --version "${version}"
+}
+
+
+NAMESPACE_SUFFIX=${RANDOM}
+NAMESPACE="cartographer-${NAMESPACE_SUFFIX}"
+kubectl create ns ${NAMESPACE}
+
+kubectl apply -n ${NAMESPACE} --filename "${MY_DIR}"/testdata.yaml
+
+for sleep_duration in {1..10}; do
```

Comment:
can we check this something like 
`kubectl wait --for=condition=Ready pod volume-test -n ${NAMESPACE}`
i don’t know kubectl wait work for config-map or not just a ask if possible as this will be  fail and print appropriate logs too on failing.

Reply:
that's a good suggestion but indeed, I don't think there's some command we could use (like `wait` as you suggested or something else similar to `rollout`) though

in this case, we're essentially just waiting for the object to exist (if at some point it exists, that means that cartographer is in good shape as it was able to create it according to the supply chain), so even if configmaps had conditions, `k wait` would still fail early as the object wouldn't initially exist

Reply:
Makes Sense Thanks @cirocosta approving and merging it.